common/thorhelper/roxiehelper.hpp

```diff
@@ -104,6 +104,7 @@ typedef enum { heapSortAlgorithm, insertionSortAlgorithm,
               mergeSortAlgorithm, spillingMergeSortAlgorithm,
               parallelMergeSortAlgorithm, spillingParallelMergeSortAlgorithm,
               tbbQuickSortAlgorithm, tbbStableQuickSortAlgorithm,
```

Comment:
Are tbbQuickSort and tbbStableQuickSort not parallel?

Perhaps some comments on this enum describing what is what would be helpful?

What algo is used if they just indicate they want a parallel sort without being specific about which one (can they do that?)